pkg/cache/shares.go

```diff
@@ -63,19 +58,15 @@ var (
 func AddSharedConfigMap(share *sharev1alpha1.SharedConfigMap) {
 	br := share.Spec.ConfigMapRef
 	key := BuildKey(br.Namespace, br.Name)
-	klog.V(4).Infof("AddSharedConfigMap key %s", key)
-	obj, ok := configmaps.Load(key)
-	if obj != nil && ok {
-		cm := obj.(*corev1.ConfigMap)
-		configmapsWithShares.Store(key, cm)
+	klog.V(4).Infof("AddSharedConfigMap share %s key %s", share.Name, key)
+	cm := client.GetConfigMap(br.Namespace, br.Name)
+	if cm != nil {
 		// so this line build a map with a single entry, the share from this event, and then
 		// applies the function(s) supplied by the CSI volume code in order to make changes based
 		// on this event
+		klog.V(4).Infof("AddSharedConfigMap share %s key %s start range", share.Name, key)
 		shareConfigMapsUpdateCallbacks.Range(buildRanger(buildCallbackMap(share.Name, share)))
-		//NOTE we do not store share in shareConfigMaps unless the backing resource is available
-		shareConfigMaps.Store(share.Name, share)
-	} else {
-		sharesWaitingOnConfigmaps.Store(share.Name, share)
+		klog.V(4).Infof("AddSharedConfigMap share %s key %s end range", share.Name, key)
```

Comment:
This looks like the same message that is logged if it succeeds. Should this be a `Warning` at least and a different message?

Reply:
this is some left over debug bracket tracing .... don't need it I'll just remove, thanks..